09_maps/main.go

```diff
@@ -85,6 +97,14 @@ func main() {
 	// - "height" -> 180
 	// - "weight" -> 70
 	// Print the map and its length.
+	var person map[string]string = map[string]string{}
+	person["name"] = "John"
+	// TODO: add rest of the values as string/int pair instead of string/string
+	person["age"] = "25"
+	person["height"] = "180"
+	person["weight"] = "70"
+	fmt.Println(person)
+	fmt.Println(len(person))
```

Comment:
@nobe4 Should I use `interface{}` in order to be able to create a map with mixed key value pairs?

Reply:
In theory yes, but let's not talk about `interface{}` for now.
Use strings everywhere for this exercise :)

Reply:
Actually, I'll need to update my exercises, they have some mixed values.
Sorry about that

Reply:
Okay, sounds good.
I was struggling a bit how to proceed due to that. Especially later with the multiplication...

Reply:
I've pushed changes to the master branch, please merge it into your branch to have (hopefully) less confusing exercises.